migrate.py

```diff
@@ -225,9 +226,13 @@ def rewrite_doc_fragments(mod_fst, collection, spec, namespace):
 def rewrite_imports(mod_fst, collection, spec, namespace):
     """Rewrite imports map."""
     plugins_path = ('ansible_collections', namespace, collection, 'plugins')
+    tests_path = ('ansible_collections', namespace, collection, 'tests')
+    unit_tests_path = tests_path + ('unit', )
+    unit_tests_path = ('unit', )  # hack
```

Comment:
Currently, `ansible-test` doesn't work with absolute imports from collections. `pytest` crashes with `ImportError` on the test collection stage, usually when scanning `conftest` files. But it adds `tests/` to `PYTHONPATH` so it works.

Reply:
### Works:
* `import test_migrate_ns.yum_collection.tests.unit.compat.mock`
* `import unit.compat.mock`

### Doesn't work:
* `import ansible_collections.test_migrate_ns.yum_collection.tests.unit.compat.mock`

Reply:
(except in `conftest.py`)